### ISSUE TITLE
some items kick player from server

### ISSUE THREAD

**Issue by imbasicallyalex** [KaloyanKKS/FoodModPlus#2]
a few of the food items like fries kick the player from a server for some reason?

**Comment by 0xKY** [comment]
Yeah this is very much possible as I haven't tested the mod in a server environment. I haven't really been working on this mdo recently but I might try to still fix this
